cli/src/main/kotlin/commands/UploadResultToSw360Command.kt

```diff
@@ -89,9 +107,27 @@ class UploadResultToSw360Command : CliktCommand(
         getProjectWithPackages(ortResult).forEach { (project, pkgList) ->
             val linkedReleases = pkgList.mapNotNull { pkg ->
                 val name = createReleaseName(pkg.id)
-                sw360ReleaseClient.getSparseReleaseByNameAndVersion(name, pkg.id.version)
+                var release = sw360ReleaseClient.getSparseReleaseByNameAndVersion(name, pkg.id.version)
                     .flatMap { sw360ReleaseClient.enrichSparseRelease(it) }
                     .orElseGet { createSw360Release(pkg, sw360ReleaseClient) }
+
+                if (attachSources) {
+                    val tempDirectory = createTempDirectory(pkg.id.toCoordinates()).toFile()
```

Comment:
Please use `createOrtTempDir()` instead which uses an ORT-specific prefix automatically.

Reply:
`toCoordinates()` is not safe to use as part of a directory name as it contains colons. Please use `pkg.id.toPath()` instead, similar to like in the `DownloaderCommand`:

https://github.com/oss-review-toolkit/ort/blob/768a585b1ccf72cbd952fe7f95216a15d11c947c/cli/src/main/kotlin/commands/DownloaderCommand.kt#L244